cli/src/nonce.rs

```diff
@@ -325,6 +338,21 @@ pub fn parse_withdraw_from_nonce_account(
     })
 }
 
+pub(crate) fn parse_upgrade_nonce_account(
+    matches: &ArgMatches<'_>,
+) -> Result<CliCommandInfo, CliError> {
+    let nonce_account = pubkey_of(matches, "nonce_account_pubkey").unwrap();
+    let memo = matches.value_of(MEMO_ARG.name).map(String::from);
+    Ok(CliCommandInfo {
+        command: CliCommand::UpgradeNonceAccount {
+            nonce_account,
+            memo,
+        },
+        // TODO: need signers.
+        signers: CliSigners::default(),
```

Comment:
This is fine because if `CliSigners` is empty, we fall back to the wallet keypair here: https://github.com/solana-labs/solana/blob/cbb0f07d543f424e8c90ac99f23ecea1a66b9327/cli/src/main.rs#L183

Reply:
removed todo